src/JsonGraphqlForm/JsonGraphqlForm.tsx

```diff
@@ -14,7 +14,7 @@ const Query = getComponent<QueryProps>('GraphqlQuery');
 export type mapFormValuesToMutationVariablesFn<Values = any> = (values: Values) => any;
 export type mapQueryDataToInitialValuesFn<Values = any> = (data: any) => Values;
 
-export type JsonGraphqlFormOnErrorFn = (error: ApolloError) => void;
+export type JsonGraphqlFormOnErrorFn<Values = FormikValues> = (error: ApolloError, values: Values) => void;
```

Comment:
can you make this interface similar to `JsonGraphqlFormOnSuccessFn`? By adding `actions` param as well? Does this make sense?

Reply:
the actions param has been added  in Error callback as success callback